File: problemas/iniciante/1065/1065.php
<?php
$pares = 0;

for($i = 0; $i < 5; $i++){
    $entrada = trim(fgets(STDIN));
    sscanf($entrada, "%d", $numero);

    if($numero % 2 == 0){
        $pares += 1;
    }
}

echo $pares . " valores pares" . PHP_EOL;
?>

File: problemas/iniciante/1065/1065.scala
object Main {
  def main(args: Array[String]): Unit = {
    var pares = 0
    for(i <- 1 to 5){
        val numero = io.StdIn.readLine().toInt
        if(numero % 2 == 0){
            pares += 1
        }
    }

    printf("%d valores pares\n", pares)
  }
}
